packages/cli/package-lock.json

```diff
@@ -4,4680 +4,6767 @@
 	"lockfileVersion": 1,
 	"requires": true,
 	"dependencies": {
-		"jest": {
-			"version": "25.5.4",
-			"resolved": "https://registry.npmjs.org/jest/-/jest-25.5.4.tgz",
-			"integrity": "sha512-hHFJROBTqZahnO+X+PMtT6G2/ztqAZJveGqz//FnWWHurizkD05PQGzRZOhF3XP6z7SJmL+5tCfW8qV06JypwQ==",
+		"@babel/code-frame": {
```

Comment:
you can discard the one-time diff from your first `npm link` here

Reply:
Done.